### PR TITLE
Add message to visitors

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

It would look like this

Screen Shot 2022-04-21 at 21 55 52

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by knanao — COMMENTED** [review]
I LOVE IT!
/lgtm

File: web/src/index.tsx [hunk 1]
```diff
@@ -19,6 +19,27 @@ async function run(): Promise<void> {
...
+appreciate your support. We seek for any kind               ▄▓▓▓▄ ▀▓▓▀▓▓▓▓▓▓▓▓
+of contributions and feedback. If you feel                 ▐▓▓▓▓▓▓       ╟▓▓▓▓▓▌
+interested, feel free to open up issues or PRs.            ▓▓▓▓▓▓         ▓▓▓▓▓▌
+                                                           ▓▓▓▓▓▓▄        ╔φ, └└
+The PipeCD official links:                                  ▓▓▓▓▓▓▀     ╠▒▒▒▒▒╠
+  Documentation: https://pipecd.dev                           ▀▓▌ ╓▓▓▓▌ ╚▒╠^
+  Github: https://github.com/pipe-cd/pipecd                     ▓▓▓▓▓▓▓▓,
+  Twitter: https://twitter.com/pipecd_dev                        ╙▓▓▓▓▓▓▓▓µ
+                                                                  └▓▓▓▓▓▓▓▌
+Love to contribute to PipeCD? we're HIRING, so                      └▀▓▓▓╙
```
Comment (nghialv) [inline web/src/index.tsx:36]:
```suggestion
Love to contribute to PipeCD? We're HIRING, so                      └▀▓▓▓╙
```

**Comment by nghialv** [comment]
Love it!
Just left a nit. 💯

**Comment by nghialv** [comment]
Nice.
/approve

**Pull request merged**